lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -18,6 +18,18 @@ class Console::CommandDispatcher::Core
 
   include Console::CommandDispatcher
 
+  METERPRETER_TRANSPORT_SSL   = 0
```

Comment:
These constants should all be in `Rex::Post::Meterpreter::ClientCore` instead of here.